docs/utils.rst

```diff
@@ -76,6 +76,12 @@ Molecular Utilities
 .. autoclass:: deepchem.utils.rdkit_util.MoleculeLoadException
   :members:
 
+.. autoclass:: deepchem.utils.molecule_graph.MoleculeGraphData
```

Comment:
Could you move this over to `docs/dataclasses.rst`? That way it will be alongsize `ConvMol` and `WeaveMol` in the documentation

Reply:
Sorry, I mistake the location 🙇‍♂️  Now, I fixed!